app/views/content/non-uk-teachers/find-a-teaching-job-as-a-non-uk-citizen.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Find a teaching job as a non-UK citizen"
+subcategory: If you want to train to teach
+description: |-
+  Learn more about applying for teaching jobs after you qualify.
+related_content:
+    Train to teach in England as an international student: "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
+    Teacher pay and benefits in England : "/is-teaching-right-for-me/teacher-pay-and-benefits"
+    A day in the life of a teacher in England : "/a-day-in-the-life-of-a-teacher"
+promo_content:
+navigation: 20.60
+navigation_title: Find a teaching job as a non-UK citizen
+navigation_description: How to successfully apply for your first teaching job.
+date: "2021-05-27"
+image: "static/images/content/hero-images/0002.jpg"
+backlink: "../../"
+keywords:
+  - International
+  - Teaching
+  - Education
+  - Schools
+  - Teacher
+  - Overseas
+  - Foreign
+  - Non-UK
+  - Visa
+  - Skilled Worker visa
+  - Immigration
+  - Student Visa
+  - Graduate Visa
+  - Youth Mobility Scheme
+  - Family Visa
+  - UK Ancestry Visa
+  - Refugees
+  - Settlement Scheme
+  
+---
+
+English [qualified teacher status](https://getintoteaching.education.gov.uk/train-to-be-a-teacher/what-is-qts) allows you to teach in any type of school in England.
```

Comment:
@laurahermionetennant92 Internal links shouldn't have the full domain - this is causing a build error